packages/addresses/src/address.ts

```diff
@@ -55,18 +56,22 @@ export function assertIsAddress(putativeAddress: string): asserts putativeAddres
             // Highest address (32 bytes of 255)
             putativeAddress.length > 44
         ) {
-            throw new Error('Expected input string to decode to a byte array of length 32.');
+            throw new SolanaError(SOLANA_ERROR__INVALID_ADDRESS, {
```

Comment:
‘Invalid’ can mean _so_ many things where an address is concerned, but ‘malformed’ would be more to the point here.